### PR TITLE
Show subtree issues

### PR DESCRIPTION
**Pull request by dadaa**

### CODE REVIEW

File: extension/sidebar-pane.html [hunk 1]
```diff
@@ -121,9 +121,13 @@
     <script type="module" src="sidebar-pane.js"></script>
   </head>
   <body>
-    <main>
+    <section id="selected">
       <header>Issues at selected node</header>
```
Comment (captainbrosset) [inline extension/sidebar-pane.html]: Proposal to rephrase this : "Issues for the selected element"

File: extension/sidebar-pane.html [hunk 2]
```diff
@@ -121,9 +121,13 @@
...
-      <section></section>
-    </main>
+      <ul></ul>
+    </section>
+    <section id="subtree">
+      <header>Issues at selected node</header>
```
Comment (captainbrosset) [inline extension/sidebar-pane.html]: Proposal to rephrase this: "Issues for the selected element's subtree"

File: extension/sidebar-pane.js [hunk 3]
```diff
@@ -29,14 +36,47 @@ async function _update() {
...
+  const declarationBlocks = await browser.experiments.inspectedNode.getStylesInSubtree();
+  for (const { declarations } of declarationBlocks) {
+    issues.push(
+      ..._webcompat.getCSSDeclarationBlockIssues(declarations, _targetBrowsers));
+  }
+
+  _render(issues, ulEl);
+}
+
+function _isNormalElement({ nodeType, isCustomElement }) {
```
Comment (captainbrosset) [inline extension/sidebar-pane.js]: I think calling this `_isValidElement` would be better for our purpose.

**Review by captainbrosset — APPROVED** [review]

**Comment by dadaa** [comment]
Thank you very much for reviewing!
I'll address all of your points.

**Pull request merged**